server/routes/findbyid.js

```diff
@@ -10,6 +10,11 @@ module.exports = function( req, res ){
     return !isNaN( id );
   });
 
+  var lang;
+  if( req.query.lang && req.query.lang.length === 3 ){
+    lang = req.query.lang;
```

Comment:
I forget, doesn't the 'lang' parameter need to be in a specific case (upperCase/lowerCase)?

Reply:
Yes `lang` must match the iso639-2T code in lower case

Reply:
Can we please transform the `req.query` to lowercase, so that the interface works for both uppercase and lowercase inputs?

Reply:
Oh, oops, looks like you just coped by shoddy code :)

I think we should `toLowerCase` all the places we deal with this?